src/script/squirrel_helper.hpp

```diff
@@ -15,6 +15,7 @@
 #include "../economy_type.h"
 #include "../string_func.h"
 #include "squirrel_helper_type.hpp"
+#include <3rdparty/squirrel/include/squirrel.h>
```

Comment:
Why is this necessary?

Reply:
It isn't; but `kdevelop` insisted that it was. I'll remove it.